<launch>
   <arg name="is_test" default="false" />
  
   <node pkg="rfd900_bridge" type="pc.py" name="rfd900_bridge_pc" output="screen">
   <param name="rfd900_port" value="/dev/ttyUSB0" />
   <remap from="tf_rfd" to="/tf" unless="$(arg is_test)" />
   </node> 
</launch>